### DISCUSSION TITLE
pcat_nit.cpp: machine has no sound or is it unemulated?

### DISCUSSION THREAD

**Discussion by FlykeSpice** [mamedev/discussions#164] [General]
This is a random machine I came across playing TouchStar Bonanza, on the frontend listing it's flagged red with a pop-up warning saying sound emulation is unimplemented.

Peeking at the source code, there is no mention on the notes that this machine has any sound and it's flagged MACHINE_NO_SOUND.

I also tried to find footage of this machine on the internet but its game names are too generic to yield any useful result.

**Comment by angelosa** [comment]
Can only speculate here:
1. there's a DAC sound chip connected to the parallel port among the other JAMMA inputs (there's some skipped initialization at POST for a latch r/w);
2. game just uses the internal buzzer for sounds;
3. game overrides buzzer like on Filetto with UM5100 sound chip (unlikely, no data in the game roms beyond a MS-DOS 3.3 with drivers and PCX images);
4. there's no sound at all;
